modcc/symdiff.cpp

```diff
@@ -266,7 +267,13 @@ class SymPDiffVisitor: public Visitor, public error_stack {
 // from the given one with constant simplifications.
 
 long double expr_value(Expression* e) {
-    return e && e->is_number()? e->is_number()->value(): NAN;
+#ifdef __FAST_MATH__
```

Comment:
I understand that the reason for this patch is to get something working for the KNL workshop.

For the workshop can you cross compile with modcc compiled without the fast-math flags.